bin/pytorch_inference/Main.cc

```diff
@@ -162,13 +166,11 @@ int main(int argc, char** argv) {
         return EXIT_FAILURE;
     }
 
-    if (isLogFileNamedPipe) {
-        if (ml::core::CLogger::instance().reconfigureLogToNamedPipe(
-                logFileName, cancellerThread.hasCancelledBlockingCall()) == false) {
-            LOG_FATAL(<< "Could not reconfigure logging");
-            cancellerThread.stop();
-            return EXIT_FAILURE;
-        }
+    if (ml::core::CLogger::instance().reconfigure(
```

Comment:
The `evaluate.py` script does not set the `logPipe` argument so all logging is done to std out which is quite useful at this stage of development. 

I think we still need to protect the call to reconfigure with `if (logFileName.empty() == false)` at least while we are using `evaluate.py`

Reply:
That protection is already in `CLogger::reconfigure()`.  As the code stands here it will default to logging to stderr, just like the other ML processes like `autodetect`.